engine/config/locales/es.yml

```diff
@@ -45,7 +45,7 @@ es:
         last_update_html: Última actualización <time id="page-updated-at" datetime="%{time}">%{time}</time>
         wording: "¡Recuerda, también tú estás haciendo un buen trabajo!"
       navbar:
-        cron_schedules: Tareas Programadas
+        cron_schedules: Horarios
```

Comment:
I know a little Spanish, but not in a programming context. Can someone verify this?

Reply:
Using "cron", it doesn't need to be translated 😉